roles/change-paths/tasks/main.yml

```diff
@@ -0,0 +1,167 @@
+---
+# Ansible role for changing paths after a site has been uploaded.
+# https://github.com/SU-SWS/ansible-playbooks
+# ==================================================================
+#
+# This role runs a number of tasks to transform the database in such a way that
+# images and absolute paths will work in the new ACSF environment. If any part
+# of this role fails for a site, restart from the tag "download-site".
+#
+# INPUTS:
+#   acsf_site_name
+#   inventory_hostname
+#   sitefactory_environment
+#   drush_environment
+#   server_alias
+#   site_prefix
+#   stack
+#   server
+#   dept_site
+#   wait_time
+#
+# OUTPUTS:
+#   file_public_path
+#
+# ALTERNATIVE ROLES:
+# --
+#
+# REQUIREMENTS:
+# --
+#
+# KNOWN ISSUES:
+# --
+
+##############################################################################
+# Database surgery: Find and replace anything that the user may have entered #
+# that references the old site.                                              #
+# See https://stanfordits.atlassian.net/browse/SITES-248 for details.        #
+##############################################################################
+
+- name: Set sites_service variable
+  set_fact:
+    sites_service: "{% if site_prefix=='dp' %}people{% elif site_prefix=='ds'%}sites{% endif %}"
+
+# Found example of a site with a vhost, hard linking to images by non-vhost path.
+- name: Set absolute vhost path from sites service
+  set_fact:
+    absolute_vhost_path_sites_service: "{{ vhost }}.stanford.edu"
+
+- name: Set absolute non-vhost path from sites service
+  set_fact:
+    absolute_path_sites_service: "{{ sites_service }}.stanford.edu/{{ inventory_hostname }}"
+
+- name: Set absolute path for testing
+  set_fact:
+    absolute_path_testing: "{% if vhost is defined %}{{ vhost }}{% else %}{{ acsf_site_name }}{% endif %}-{{ acsf_environment }}.cardinalsites.stanford.edu" 
+
+- name: Set absolute path for production
+  set_fact:
+    absolute_path_prod: "{% if vhost is defined %}{{ vhost }}{% else %}{{ acsf_site_name }}{% endif %}.stanford.edu"
+
+- name: Set new absolute path based on destination
+  set_fact:
+    new_absolute_path: "{% if acsf_environment == '' %}{{ absolute_path_prod }}{% else %}{{ absolute_path_testing }}{% endif %}"
+
+- name: Debug absolute path for site
+  debug:
+    msg: "Absolute path: {{ new_absolute_path }}"
+
+- name: Set drush sar command
```

Comment:
Let's call this "Set drush alias" and call the variable `drush_alias`

Reply:
Looks like you changed this.